python/cudf/cudf/tests/test_rank.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from itertools import chain, combinations_with_replacement, product
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from cudf.core import DataFrame
+from cudf.tests.utils import assert_eq
+
+
+class TestRank:
```

Comment:
Any reason for putting these tests in a class?

Reply:
no reason except thought of sharing index, col1, col2 among series and dataframe tests. dataframe test is disabled for now.

Reply:
Thanks. We could do that by defining them at the module level as well. Let's avoid the additional complexity of using classes for tests unless absolutely required.